--- mate-sensors-applet-1.6.1/configure.ac.orig	2014-01-01 14:55:07.000000000 +0100
+++ mate-sensors-applet-1.6.1/configure.ac	2014-01-04 08:34:08.642776489 +0100
@@ -91,7 +91,7 @@
 	AC_SUBST(LIBATASMART_LIBS)
 fi
 
-AM_CONDITIONAL(UDISKS, test -n "$DBUSGLIB_CFLAGS" -a -n "$LIBATASMART_CFLAGS")
+AM_CONDITIONAL(UDISKS, test -n "$DBUSGLIB_LIBS" -a -n "$LIBATASMART_LIBS")
 
 # support for libnotify
 LIBNOTIFY_CFLAGS=
